std_msgs/String classifierName
std_msgs/String[] item_ids
std_msgs/Float64[] confidences